modules/running-distributed-data-science-workloads-disconnected-env.adoc

```diff
@@ -0,0 +1,49 @@
+:_module-type: PROCEDURE
+
+[id="running-distributed-data-science-workloads-disconnected-env_{context}"]
+= Running distributed data science workloads in a disconnected environment
+
+[role='_abstract']
+To run a distributed data science workload in a disconnected environment, you must be able to access a Ray cluster image, and the data sets and Python dependencies used by the workload, from the disconnected environment.
+
+.Prerequisites
+* You have logged in to {openshift-platform} with the `cluster-admin` role.
+* You have access to the disconnected data science cluster.
+* You have installed {productname-long} and created a mirror image as described in link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_openshift_ai_self-managed_in_a_disconnected_environment[Installing and uninstalling {productname-short} Self-Managed in a disconnected environment].
+* You can access the following software from the disconnected cluster:
+** A Ray cluster image
+** The data sets and models to be used by the workload
+** The Python dependencies for the workload, either in a Ray image or in your own Python Package Index (PyPI) server that is available from the disconnected cluster
+* You have logged in to {productname-long}.
+* You have created a data science project.
+
+.Procedure
+. Configure the disconnected data science cluster to run distributed workloads as described in link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects/working-with-distributed-workloads_distributed-workloads#configuring-distributed-workloads_distributed-workloads[Configuring distributed workloads].
+. Ensure that the notebook or pipeline includes a link to a Ray cluster image that can be accessed from the disconnected environment:
```

Comment:
```suggestion
. During the instantiation of a Ray Cluster via the `ClusterConfiguration` code, ensure that the notebook or pipeline includes a link to a Ray cluster image that can be accessed from the disconnected environment:
```

Here is an [example of exactly](https://github.com/project-codeflare/codeflare-sdk/blob/62be50e23e7ecba074e540c4715f0dd9f4ca33a0/demo-notebooks/guided-demos/2_basic_jobs.ipynb#L59-L70) what this is along with a community demo image on line 69 which is the image we use for our internal testing hosted in our own public registry.